metricbeat/module/elasticsearch/index/data.go

```diff
@@ -179,7 +182,7 @@ type bulkStats struct {
 }
 
 func eventsMapping(r mb.ReporterV2, httpClient *helper.HTTP, info elasticsearch.Info, content []byte, isXpack bool) error {
-	clusterStateMetrics := []string{"routing_table"}
+	clusterStateMetrics := []string{"routing_table", "metadata"}
```

Comment:
Since metadata can be quite big depending on how many indices exist, I would add a `filter_path` to this query to only retrieve what's necessary. The following one should be a good starting point: `filter_path=routing_table,**.settings.index.**._tier_preference,**.settings.index.creation_date`

Reply:
That's a good idea, but I think we should leave it for another PR